executor/show.go

```diff
@@ -488,6 +490,25 @@ func (e *ShowExec) fetchShowCreateTable() error {
 	return nil
 }
 
+// Compose show create database result
+func (e *ShowExec) fetchShowCreateDatabase() error {
+
+	db, ok := e.is.SchemaByName(e.DBName)
+	if !ok {
+		return infoschema.ErrDatabaseNotExists.Gen("Unknown database '%s'", e.DBName.O)
+	}
+
+	var buf bytes.Buffer
+	buf.WriteString(fmt.Sprintf("CREATE DATABASE `%s`", db.Name.O))
+	if s := db.Charset; len(s) > 0 {
+		buf.WriteString(fmt.Sprintf(" DEFAULT CHARSET=%s", s))
```

Comment:
It prints `/*!40100 DEFAULT CHARACTER SET utf8 */` in the MySQL. Do we need
 to be the same as MySQL?